web/src/components/Collapsible.js

```diff
@@ -31,7 +30,11 @@ class Collapsible extends Component {
     const { open: openState } = this.state;
 
     const isOpen = onChange ? openProp : openState;
-    const contentId = `collapsible-${kebabCase(title)}`;
+    const replaceSpaces = / /g;
+    const removeColon = /:/g;
+    const removeCarat = /›/g;
```

Comment:
I could group these together, but I sort of liked naming them explicitly because regexes are awful.